scheduler/src/cook/mesos/api.clj

```diff
@@ -1560,15 +1560,15 @@
                                                                           (into [])))}]
                          :else
                          (let [time-until (ratelimit/time-until-millis! job-submission-rate-limiter user)
-                               in-debt? (zero? time-until)]
+                               in-debt? (not (zero? time-until))]
                            ; Always log if we're in debt & rejecting.
                            (when in-debt?
                              (log/warn (str "User " user " is inserting too quickly. Not allowed to insert for "
-                                            (-> time-until t/millis t/in-seconds) " seconds")))
+                                            (/ time-until 1000.0) " seconds")))
```

Comment:
Why did you revert this?

Reply:
Wrong round off. I did some evaluation and got messages saying:
"Not allowed to inset for 0 seconds". That's not nice UI, when the delay is ".266 seconds".  I didn't see an option in `clj-time.core` to make it do the conversion with rounding up.